firmware/Core/Src/telecommands/flash_bank_telecommands_defs.c

```diff
@@ -0,0 +1,85 @@
+#include "telecommands/flash_bank_telecommands_defs.h"
+#include "flash_bank/flash_bank.h"
+#include "stm32l4xx_hal.h"
+#include "telecommands/telecommand_args_helpers.h"
+
+#include <stdio.h>
+#include <string.h>
+
+/// @brief Write data to the internal flash bank starting from address 0x08000000
+/// @param args_str
+/// - Arg 0: The data to write as a string
+/// @note This telecommand is only for testing purposes, it is purposfully not fully fleshed out
+/// as there is no intention on using this. Update as needed
+/// @note Currently, this is only writes all data given as a string.
+/// Thus, it is difficult to write integers directly
+uint8_t TCMDEXEC_flash_bank_write(const char *args_str, TCMD_TelecommandChannel_enum_t tcmd_channel, char *response_output_buf, uint16_t response_output_buf_len)
+{
+    const uint32_t res = Flash_Bank_Write(Flash_Partitions_FLASH_BANK2, (uint8_t *)args_str, strlen(args_str));
+    if (res != 0)
+    {
+        snprintf(response_output_buf, response_output_buf_len, "Error: %lu", res);
+        return 1;
+    }
+    return 0;
+}
+
+/// @brief Read data from the internal flash bank starting from address 0x08000000
+/// @param args_str
+/// - Arg 0: The number of bytes to read as a uint64_t
+uint8_t TCMDEXEC_flash_bank_read(const char *args_str, TCMD_TelecommandChannel_enum_t tcmd_channel, char *response_output_buf, uint16_t response_output_buf_len)
```

Comment:
Doesn't this function need an address or an offset to read from? 

What format does it display the value in? I recommend hex.

Reply:
done